templates/register2faOtp.html

```diff
@@ -29,17 +29,53 @@ <h4 class="section-title">2. Enter the 6 digits code </h4>
                         <input class='validate otp' type='number' name='otp' id='otp' max="999999" minlength="6" maxlength="6" autofocus/>
                         <label for='otp'>Use the entry starting with "<span name="data-connection-name"></span>"</label>
                         <span class="helper-text" data-error="Must be 6 digits" data-success="right"></span>
+                        <button type="button" class="waves-effect waves-light btn" id="otp-button">Submit</button>
```

Comment:
Was there an issue that made you add a submit button here? Normally it should auto-submit once the 6 digits have been entered.

Reply:
Yes, auto submit on keypress does not work in a mobile browser.